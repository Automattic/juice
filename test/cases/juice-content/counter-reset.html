<html>
  <head>
      <style>
        p {
          color: red;
        }
      </style>
  </head>
  <body>
    <p style="counter-reset: li;">Cool!</p>
  </body>
</html>
