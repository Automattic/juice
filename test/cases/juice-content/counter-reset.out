<html>
  <head>
      
  </head>
  <body>
    <p style="color: red; counter-reset: li;">Cool!</p>
  </body>
</html>
